Sources/AllOfSchema.swift

```diff
@@ -3,6 +3,14 @@ import ObjectMapper
 public struct AllOfSchema {
     public let metadata: Metadata
     public let subschemas: [Schema]
+    
+    /// Determines whether or not the schema should be considered abstract code
```

Comment:
I think you can omit 'code generation purposes'. I don't want to make assumptions about what this library should be used for. Someone could use it for linting documents or building a UI for swagger docs.